gradle/dependencies.gradle

```diff
@@ -0,0 +1,38 @@
+ext.versions = [
+    kotlin: '1.1.51',
```

Comment:
Should we bump it?

Reply:
We can't because of the gradle-intellij-plugin issue.